gensim/models/wrappers/dtmmodel.py

```diff
@@ -97,8 +97,8 @@ def __init__(self, dtm_path, corpus=None, time_slices=None, mode='fit', model='d
              Max em optimization iterations in LDA.
         alpha : int, optional
             Hyperparameter that affects sparsity of the document-topics for the LDA models in each timeslice.
-        top_chain_var : int, optional
-            Hyperparameter that affects.
+        top_chain_var : float, optional
+            Hyperparameter that determines how similar topics would be over multiple timeslice.
```

Comment:
Can we be more explicit? What does this mean, what value should I pick as a user?